tests/Zend/DebugTest.php

```diff
@@ -51,8 +51,8 @@ public function testDebugDump()
         $data = 'string';
         $result = Zend_Debug::Dump($data, null, false);
         $result = str_replace(array(PHP_EOL, "\n"), '_', $result);
-        $expected = "__string(6) \"string\"__";
-        $this->assertEquals($expected, $result);
+        $expected = "__.*string\(6\) \"string\"__";
```

Comment:
With newer versions of Xdebug, the full file path and line number is in this output. Using a regex condition to ignore that bit if present, but still check for the other strings.